tekton/release-cheat-sheet.md

```diff
@@ -25,26 +32,26 @@ the pipelines repo, a terminal window and a text editor.
       - name: url
         value: https://github.com/tektoncd/pipeline
       - name: revision
-        value: # Example: 33e0847e67fc9804689e50371746c3cdad4b0a9d
+        value: # The commmit you selected in the last step, e.g. 33e0847e67fc9804689e50371746c3cdad4b0a9d
     ```
 
-4. `kubectl apply -f tekton/resources.yaml`
+1. Apply file you just made to the dogfooding cluster: `kubectl apply -f your-pipeline-resource-file.yaml`
 
-5. Create environment variables for bash scripts in later steps.
+1. Create environment variables for bash scripts in later steps.
 
     ```bash
-    TEKTON_VERSION=# Example: v0.11.2
-    TEKTON_RELEASE_GIT_RESOURCE=# Example: tekton-pipelines-v0-11-2
-    TEKTON_IMAGE_REGISTRY=gcr.io/tekton-releases
+    export TEKTON_VERSION=# Example: v0.11.2
```

Comment:
since you are updating it, how about pointing it to a new release instead of patch i.e. replacing `.2` with `.0` in `v0.11.2`